settings.gradle.kts

```diff
@@ -16,3 +16,12 @@ include(
     "ml-completion",
     "intellij-toml"
 )
+
+// Configure Gradle Build Cache. It is enabled in `gradle.properties` via `org.gradle.caching`.
+buildCache {
+    local {
+        isEnabled = System.getenv("CI") == null
+        directory = File(rootDir, "build/build-cache")
```

Comment:
It doesn't work with `clean` task properly because `clean` task remove `build` dir and you get
```
Execution failed for task ':common:compileKotlin'.
> Failed to store cache entry for task ':common:compileKotlin'
```

And I don't know how to avoid it because all code in `settings.gradle` executed before any task execution and at the same time it seems caching code doesn't call `mkdirs` for its `directory`

Maybe it's worth just using a separate `build-cache` dir and mark it as excluded dir as we do with `deps` dir?

Reply:
I moved `build-cache` directory to the top level and configured `clean` task to also clean this directory